lib/matplotlib/axes/_axes.py

```diff
@@ -4938,6 +4938,12 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
             marker_obj = marker
         else:
             marker_obj = mmarkers.MarkerStyle(marker)
+        if cbook._str_equal(marker_obj.get_marker(), ","):
+            _api.warn_external(
+                "The pixel maker ',' is not supported on scatter(). Fallback "
+                "to a finite-sized square, which is not necessarily 1 pixel in "
```

Comment:
Fallback isn't a verb, but even so, I find the second sentence a bit confusing. Maybe instead
```suggestion
                "The pixel maker ',' is not supported on scatter(); using "
                "a finite-sized square instead, which is not necessarily 1 pixel in "
```